docs/metrics.md

```diff
@@ -63,6 +63,16 @@ The following metrics are added to the ping:
 This ping is sent at the end of a session (when Firefox Reality switches to the background). We usually send search and UI control metrics at the end of a session.
 
 
+This ping includes the [client id](https://mozilla.github.io/glean/book/user/pings/index.html#the-client_info-section).
+
+**Data reviews for this ping:**
+
+- <https://github.com/MozillaReality/FirefoxReality/pull/2241#issuecomment-557740258>
+
+**Bugs related to this ping:**
+
+- <https://github.com/MozillaReality/FirefoxReality/issues/2230>
+
```

Comment:
Please consider to delete  `metrics.md` and do a clean build. This change looks odd to me. If it wanna list issues, why
- issues/1609
- issues/1610
- issues/2347
- pull/1854#issuecomment-546214568
- pull/2348#issuecomment-564736919
- pull/3199#issuecomment-617938749
... etc.

doesn't be added

Reply:
This section of the documentation is only related to the bugs and data-reviews _of the ping_, which are listed in [this file](https://github.com/MozillaReality/FirefoxReality/blob/master/app/pings.yaml). The data reviews and bug for the metrics contained in the ping are still shown, as usual, linked in the table below.